.github/workflows/version_check.yml

```diff
@@ -55,56 +73,67 @@ jobs:
         id: get_curr_ocp_version
         run: |
           OCP_VERSION=$(./oc version -o json | jq '.openshiftVersion')
+          OCP_VERSION=$(sed -e 's/^"//' -e 's/"$//' <<< $OCP_VERSION)
```

Comment:
I think for better readability of the code, we can combine in a single statement 
` OCP_VERSION=$(./oc version -o json | jq '.openshiftVersion' | sed -e 's/^"//' -e 's/"$//' ) `

Reply:
Prefer to leave as-is. Do not like single line multi-commands because it one fails it is more difficult to find the culprit.